creativecommons/vocabulary
# Add GitHub Actions Workflow for Linting

## Related to
- Fixes #5  by @Vanessa082 

## Description
This PR adds a GitHub Actions workflow to implement linting for HTML, CSS, and JavaScript files in the project. The workflow runs on pushes and pull requests to the `main` branch and utilizes the `github/super-linter` action for code linting.

## Technical details
- A new GitHub Actions workflow file is created with the following specifications:
  - Triggered on push and pull request events.
  - Checks out the code base to ensure the latest changes are included.
  - Runs the Super Linter against the code base, validating the code style according to established best practices.

## Tests
To verify that this PR functions correctly:
1. Push changes to a branch and create a pull request to the `main` branch.
2. Check the Actions tab in the GitHub repository to see if the linting job runs successfully.
3. Review the results of the linting to ensure there are no errors.

## Screenshots


## Checklist




[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin




For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open-source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
